ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/README.md

```diff
@@ -14,7 +14,12 @@ Pan and zoom into the desired area, making sure the area is within the red bound
 
 ## How it works
 
-Pan and zoom into the desired area, making sure the area you want to take offline is within the red rectangle. Tap on the `Export tiles` button, and this will obtain the default parameters and call `exportTileCacheWithParameters`, which will return a job. The job must be started manually, and once started can be canceled at anytime. Once the job completes successfully, the output tile cache is added as an `ArcGISTiledLayer` to the map.
+1. Create a map and set its `minScale` to 10,000,000. Limiting the scale in this sample limits the potential size of the selection area, thereby keeping the exported tile package to a reasonable size.
+2. Create an `ExportTileCacheTask`, passing in the URI of the tiled layer.
```

Comment:
Qt doesn't really use the term URI, so we convert to URL throughout our documentation

```suggestion
2. Create an `ExportTileCacheTask`, passing in the URL of the tiled layer.
```